store/in_memory_alert_def_store.go

```diff
@@ -6,7 +6,7 @@ import (
 
 // InMemoryAlertStore - concrete implementation of the AlertDefStore interface
 type InMemoryAlertStore struct {
-	data []models.AlertDef
+	Data []models.AlertDef
```

Comment:
ideally, this should not be public. access to this should be via methods defined on this struct.